pygen/pygen_src/isa/rv32i_instr.py

```diff
@@ -0,0 +1,67 @@
+from pygen_src.riscv_defines import *
+from pygen_src.riscv_instr_pkg import *
+# LOAD instructions
+DEFINE_INSTR(riscv_instr_name_t.LB,     riscv_instr_format_t.I_FORMAT, riscv_instr_category_t.LOAD, riscv_instr_group_t.RV32I, g = globals())
```

Comment:
two nits:
1) please indent the second line of each function call to match the opening parenthesis
2) since the definition of `DEFINE_INSTR` in `riscv_defines.py`, I don't think you need to explicitly pass in `g = globals()` in each instruction definition call, right?